broker-daemon/relayer/relayer-client.js

```diff
@@ -97,6 +97,8 @@ class RelayerClient {
 
       try {
         const watcher = this.orderbookService.watchMarket(params)
+        // this flag tells us if we are the in the middle of migrating the db
+        let migrating = false
```

Comment:
[nit] use `null` or let the value be undefined. It might be weird to set this as a boolean and then assign it to a function later in the method